includes/blueprints/import.php

```diff
@@ -296,6 +296,13 @@ function import_terms( array $post_terms ) {
 				'description' => $term['description'] ?? '',
 			];
 
+			// Continue if the term already exists.
+			$term_already_exists = term_exists( $term['name'], $term['taxonomy'] );
```

Comment:
I _think_ we need to pass the third argument to `term_exists()` to account for hierarchical terms. Something like this (untested):
```suggestion
			$term_already_exists = term_exists( $term['name'], $term['taxonomy'], $term['parent'] );
```
Note: you'd want to verify `$term['parent']` exists before using it, to avoid PHP notices. For example, note a few of our test terms lack the parent property (which is fine, but illustrates the need to check): https://github.com/wpengine/atlas-content-modeler/blob/d8461cfebe37352a6fb9e97b9340af4b50d0f4dd/tests/integration/blueprints/test-data/blueprint-good/acm.json#L169-L179